spec/models/taging_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe Taging, type: :model do
+ 
+  before(:each) do 
+    p @user = build(:user)
+    p @user1 = build(:user,email: "xyz@abc.com")
+    p @task = build(:task,user_id: @user.id)
+  end  
+ 
+  it "is valid with valid attributes", :skip_before => true do
```

Comment:
@rakesh-1010 `it` statements should generally start with `should`. You can rewrite this as:
`it "should be valid if valid attributes are provided"`